### PR TITLE
Add informative tooltips for column headers

### PR DESCRIPTION
**Pull request by sven1103**
This PR adds informative tooltips when hovering over the column headers in the
sample tracking viewer table.

### CODE REVIEW

**Comment by jenniferboedker** [comment]
I took the liberty to change the base branch to the story branch ;)

File: sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy [hunk 1]
```diff
@@ -291,6 +293,22 @@ class ProjectOverviewView extends VerticalLayout implements HasHotbar, HasTitle
 
         projectGrid.setHeightMode(HeightMode.ROW)
 
+        HeaderRow headerRow = projectGrid.getDefaultHeaderRow()
+        headerRow.getCell("Subscription").setStyleName("header-with-tooltip")
+        headerRow.getCell("Subscription").setDescription("Select a project you want to get notified via email and get daily status updates.")
```
Comment (jenniferboedker) [inline sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy:298]:
```suggestion
        headerRow.getCell("Subscription").setDescription("Select a project to get status updates per email.")
```

Comment (jenniferboedker) [reply sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy:298]: Do you think we should mention here that emails are only sent when specific statuses change?

Comment (sven1103) [reply sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy:298]: i think `updates` implies that imo

Comment (jenniferboedker) [reply sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy:298]: I would still advise here to try to keep the sentence shorter and only include as much information as needed but enough to understand. I think the information on the daily updates is not necessary.

**Review by jenniferboedker — APPROVED** [review]

**Pull request merged**